test/method/testbader.py

```diff
@@ -56,3 +57,31 @@ def test_val(self):
         refData = [6.9378, 0.3639, 0.3827]  # values from `bader` package
 
         assert_allclose(self.analysis.fragcharges, refData, atol=0.15)
+
+    def test_chgsum_hf(self):
+        """Does the sum of charges equate to the number of electrons for a simple molecule?"""
+
+        hfpath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "hf.out")
+        data = ccread(hfpath)
+        vol = volume.Volume((-6, -6, -6), (6, 6, 6), (0.2, 0.2, 0.2))
+        analysis = Bader(data, vol)
+        analysis.calculate()
+
+        self.assertAlmostEqual(numpy.sum(analysis.fragcharges), 10, delta=1)
+
+    def test_symms_benzene(self):
+        """ Do the carbons in benzene ring get assigned with roughly equal charges?"""
+
+        benzenepath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "benzene.out")
+        data = ccread(benzenepath)
+        vol = volume.read_from_cube(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), "benzene.cube")
+        )
+        assert_allclose(vol.origin, numpy.array([-5, -5, -5]), atol=0.5)
+
+        analysis = Bader(data, vol)
+        analysis.calculate()
+
+        self.assertAlmostEqual(
+            analysis.fragcharges[0:6].max(), analysis.fragcharges[0:6].min(), delta=0.5
```

Comment:
Looking at the resulting charges
```
array([4.36361172, 4.83148531, 4.66353415, 4.36361172, 4.8317697 ,
       4.66353415, 0.67225235, 0.51518601, 0.82588478, 0.67225235,
       0.51518601, 0.82588478])
```
do you know why there is such a large discrepancy?

Reply:
My understanding is that the discrepancy is due to the shape of the grid (which is a cube). Now that you commented this, I guess I could check whether adjusting the grid improves the discrepancy?

Reply:
I think just checking it by hand and leaving a code comment about the grid coarseness is enough (if that's the reason), since we don't want to introduce a finer grid in the tests themselves due to time constraints.

Reply:
I've just added a comment about this. Thanks!

Reply:
Can you word it so that it's clear `bader` is referring to the original reference program?

Reply:
Sure -- I've just made the necessary changes. Thanks.